File: ironpythoninaction/chapter14/14.1.4/Scriptable/Scriptable/Class1.cs
using System;
using System.Windows.Browser;

namespace Scriptable
{
    [ScriptableType]
    public class Scriptable
    {
        [ScriptableMember]
        public string method(string value)
        { return this.real(value); }

        public virtual string real(string value)
        { return "override me"; }
    }
}

File: ironpythoninaction/chapter14/14.1.4/ScriptableVB/ScriptableVB/Class1.vb
Imports System
Imports System.Windows.Browser

<ScriptableType()> _
Public Class Scriptable
    Public Function method(ByVal value As String) As String
        Return real(value)
    End Function
    <ScriptableMember()> _
    Public Overridable Function real(ByVal value As String) As String
        Return "override me"
    End Function
End Class